README.md

```diff
@@ -138,6 +146,55 @@ Run the below command to see what Symfony services are shared with this plugin:
 $ bin/console debug:container payplug_sylius_payplug_plugin
 ```
 
+### Template overriding
+
+This plugin override some sylius templates. 
+If you plan override them also, you should retrieve them in your application.
+
+Copy Sylius templates overridden in plugin to your templates directory (e.g templates/bundles/)
+
+   ```shell
+    mkdir -p templates/bundles/SyliusAdminBundle/
+    mkdir -p templates/bundles/SyliusShopBundle/
+    cp -R vendor/payplug/sylius-payplug-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
+    cp -R vendor/payplug/sylius-payplug-plugin/src/Resources/views/SyliusShopBundle/* templates/bundles/SyliusShopBundle/
+    ```
+
+### Assets
```

Comment:
Maybe this should be in a Plugin development doc

Reply:
Could we resolve conversation ?